leptonai/api/v1/types/deployment.py

```diff
@@ -248,6 +248,8 @@ class LeptonDeploymentState(str, Enum):
     Starting = "Starting"
     Updating = "Updating"
     Deleting = "Deleting"
+    Stopping = "Stopping"
+    Stopped = "Stopped"
     Unknown = ""
 
```

Comment:
Could you override the __new__ function in LeptonDeploymentState like follows, and see if it fixes future-proof errors?

```
def __new__(cls, value):
    if value not in cls._value2member_map_:
        warning.warn("You might be using an out of date SDK. consider updating.")
        value = cls.Unknown
    return str.__new__(cls, value)
```